app/models/generic_event/per_court_excessive_delay_not_due_to_supplier.rb

```diff
@@ -1,11 +1,9 @@
 class GenericEvent
   class PerCourtExcessiveDelayNotDueToSupplier < GenericEvent
     LOCATION_ATTRIBUTE_KEY = :location_id
-    DETAILS_ATTRIBUTES = %w[
-      subtype
-      vehicle_reg
-      ended_at
-    ].freeze
+
+    details_attributes :subtype, :vehicle_reg, :ended_at, :authorised_by, :authorised_at
```

Comment:
Where did the new `authorised_by` and `authorised_at` attributes come from? Are those intentional?

Reply:
Scratch that - those are defined in the included concern 👍